pkg/exchange/coinbase/api/v1/get_candles_request.go

```diff
@@ -17,15 +21,27 @@ type Candle struct {
 	Volume fixedpoint.Value `json:"volume"`
 }
 
-type GetCandlesResponse []Candle
+type GetCandlesResponse []RawCandle
+
+func (rc *RawCandle) Candle() *Candle {
+	values := *rc
```

Comment:
Verify that the rc is not nil and that the slice length is as expected. ( e.q. len == 6 ). Otherwise, the service will panic.